### DISCUSSION TITLE
How many Skip Transpiled Apps in the G. Play Store?

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#193] [Q&A] [answered]
I'm wondering how many Apps have made it into the Google Play Store?

I'm still developing my FIRST Skip-able project.  But looking into signing up on the Play Store... there are several hurdles:

- After getting an account - paying the $25 entry fee...
- First big hurdle is proving I'm a person - Drivers License & Power bill...
- Next hurdle - I need a REAL Physical Android phone - I've never had one...  Ordered a USED Pixel 6 $150.
- next... unknown as of now...

**Comment by marcprux** [comment, marked as answer]
We don't compile a list, but there are apps in both stores created with Skip. For example, our own [SkipUI Showcase](https://skip.tools/docs/samples/skipapp-showcase/) app.

There are many hurdles to get approved for app store distribution, both for the Apple App Store and Google Play Store. We sympathize with any frustration, but they are outside of our control.
